### ISSUE TITLE
Helm chart / app version matching?

### ISSUE THREAD

**Issue by artem-nefedov** [signalfx/splunk-otel-collector#1429]
I originally assumed that helm chart versions should always contain app version that exactly matches that of helm chart. However, just now I noticed that helm chart version `0.47.0` has `appVersion: 0.47.1`.  And `0.47.1` chart is exactly the same as `0.47.0`, besides chart version itself:

```
➜ diff -r splunk-otel-collector-0.47.{0,1}
diff -r splunk-otel-collector-0.47.0/Chart.yaml splunk-otel-collector-0.47.1/Chart.yaml
21c21
< version: 0.47.0
---
> version: 0.47.1
```

Is this normal? Maybe there's some race condition in release process?

**Comment by jvoravong** [comment]
This is normal, a couple things did happen that might seem odd.

The versioning scheme usually follows `major.minor.patch`.  The `major.minor` portion should typically match between the collector and the chart, the patch portion can be different. 

The collector v0.47.0 had a bug in it that broke part of the chart, the collector v0.47.1 fixed the bug. This is why the chart v0.47.0 was released with the collector v0.47.1.

The chart v0.47.1 only added [documentation](https://github.com/signalfx/splunk-otel-collector-chart/pull/419) about upgrading and applying a recently available bug fix. This is why you don't see any diffs between the Chart.yaml files.

**Comment by artem-nefedov** [comment]
OK, if this is actually normal, feel free to close this.
